vtex-apps/menu
# Close submenus when URL changes

#### What problem is this solving?

https://vtex-dev.atlassian.net/jira/software/projects/SF/boards/105?assignee=5d794d62656e2a0dbab2e04d&selectedIssue=SF-461

Guarantee that a submenu will close whenever the URL changes.

#### How to test it?

Compare:
https://kiwi--muji.myvtex.com/fashion---accessories/accessories/hats?__bindingAddress=www.mujionline.eu/uk
with
https://muji.myvtex.com/furniture/sofas-and-chairs/body-fit-cushion?__bindingAddress=www.mujionline.eu/uk

1) hover any menu and switch to another category
2) note that on the `kiwi` workspace, the menu closes after the URL has changed.

#### Screenshots or example usage:

n/a

#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/l3q2Kgm3KOBUE6wF2/giphy-downsized.gif)
